redis/src/connection.rs

```diff
@@ -201,6 +201,16 @@ impl IntoConnectionInfo for ConnectionInfo {
     }
 }
 
+/// Implementing IntoConnectionInfo for primitive string. Allowing a primitive string
+/// type to be converted into ConnectionInfo struct. Examples of valid strings below.
+///
+/// URL format: `redis://[<username>][:<password>@]<hostname>[:port][/<db>]`
```

Comment:
```suggestion
/// URL format: `{redis|rediss}://[<username>][:<password>@]<hostname>[:port][/<db>]`
```

Reply:
@NuclearOreo I think you made the `{redis|rediss}` change here on the wrong line.

Reply:
@jaymell Sorry for the silly mistake; I just fixed it. Multi-tasking is definitely not good for me lol